APIcall/data_collection.py

```diff
@@ -0,0 +1,41 @@
+import pandas as pd
+import requests
+import argparse
+from datetime import date
+
+def api_311(start_date, end_date, skip, limit):

```

Comment:
Do we need to give `skip` and `limit` as args here?

Reply:
Actually we do not. But, I added it there, just in case, some user wanted to change the number of records that were being fetched. Let me know if you want me to remove them.

Reply:
For now, I am removing the skip and limit args.